### DISCUSSION TITLE
how is the Windows Key defined in mapping?

### DISCUSSION THREAD

**Discussion by cxj05h** [sezanzeb/input-remapper#26] [Q&A] [answered]
thanks!

**Comment by sezanzeb** [comment, marked as answer]
EDIT: It should be possible to map the super key now via the GUI

## Mapping Start/Super to something

If you are trying to map the windows-key to something different, the application might loose its focus. It requires editing the configuration file.

1. open key-mapper and createa a new preset for your device

2. insert code 125 into ~/.config/key-mapper/presets/your-device/your-preset.json

```
{
    "macros": {},
    "mapping": {
        "1,125,1": "b"
    }
}
```

3. start key-mapper again to see "LEFTMETA" in the left column. You can also change it to map to something different in the UI now.

## Mapping any key to Start/Super

use `key_leftmeta` in the right column

**Reply by cxj05h** [reply]
oh cool thanks!

**Reply by JesseBourret-Gheysen** [reply]
The Windows Key Naming convention was not obvious to me. After scrolling through all the symbol names. Thanks for posting.
